src/arrayMethodSort.js

```diff
@@ -4,8 +4,18 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
```

Comment:
The default compare function should return a number rather than a boolean. In JavaScript, the `>` operator returns a boolean value. So, `String(a) > String(b)` will return either `true` or `false`. However, according to the specification of `Array.prototype.sort()`, the compare function should return a negative, zero, or positive value depending on the arguments. You might want to change it to `(a, b) => String(a).localeCompare(String(b))`.

Reply:
Поміняв, але тепер не проходить тест "sorts mixed case strings"